### ISSUE TITLE
QUESTION - Depending on Latest Version of Jasmine-Core

### ISSUE THREAD

**Issue by DarthOpto** [larrymyers/jasmine-reporters#166]
Any idea when this package will be using the latest 2.5.x version of jasmine-core?  This is stopping reports from working correctly as I am now getting errors from Jasmine Core when try to get reports.

```<REST OF PATH>/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:2219
      this.result.failedExpectations.push(this.expectationResultFactory(data));
                                               ^
TypeError: this.expectationResultFactory is not a function
    at Suite.onException (/Users/curtissalisbury/src/github.com/artemishealth/artemis-app/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:2219:48)
    at QueueRunner.onException (/Users/curtissalisbury/src/github.com/artemishealth/artemis-app/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:2330:28)
    at onException (/Users/curtissalisbury/src/github.com/artemishealth/artemis-app/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1924:12)
    at Timeout._onTimeout (/Users/curtissalisbury/src/github.com/artemishealth/artemis-app/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1910:11)
    at tryOnTimeout (timers.js:232:11)
    at Timer.listOnTimeout (timers.js:202:5)```

This has been fixed in jasmine-core 2.5.x. So when will this package be updated to accommodate that?

**Comment by bloveridge** [comment]
I think the correct solution for this is to remove `jasmine-core` as a dependency of `jasmine-reporters` and make it a `devDependency` instead. `jasmine-reporters` does not directly invoke anything from `jasmine-core`, neither methods on the exported global nor any methods on spec or suite objects.

`jasmine-reporters` is designed to be used with Jasmine, not to use / provide Jasmine itself. I'll shortly push up a new version that removes `jasmine` as a dependency, and you should be able to use any version of Jasmine with a compatible reporters interface. (To my knowledge, there is nothing about Jasmine 2.5 that fundamentally changes the reporter interface to break, and `jasmine-reporters` tests all pass with the updated version.)

**Comment by bloveridge** [comment]
New version without a `jasmine-core` dependency has just been published to npm as `2.2.1`

**Comment by DarthOpto** [comment]
Thank you very much.
